packages/core/src/utils.ts

```diff
@@ -31,23 +36,31 @@ export const attachIfSubscribed = (
 };
 
 /**
- * Check if the participant track is pinned.
+ * Check if the `TrackBundle` is pinned.
  */
-export function isParticipantTrackPinned(
-  trackParticipantPair: TrackParticipantPair,
+export function isTrackBundlePinned(
```

Comment:
wondering whether this should be an overloaded method that doesn't rely on `TrackBundle` as a type.

If we leave it as `isTrackPinned` and have overloads for
```
isTrackPinned(track);
isTrackPinned(participant, source);
```
that seems slightly easier to reason about

Reply:
Fine by me, but I would rather put it in a separate PR.